atlasdb-commons/src/test/java/com/palantir/util/IndexEncodingUtilsTest.java

```diff
@@ -57,129 +54,123 @@ public static Iterable<ChecksumType> data() {
         return Arrays.asList(ChecksumType.values());
     }
 
-    private static final Map<DH<String>, Long> VALUES =
-            ImmutableMap.of(dh("key1"), 42L, dh("key2"), 1337L, dh("anotherKey"), -10L);
-    private static final Set<DH<String>> KEYS = ImmutableSet.of(dh("key1"), dh("key2"), dh("anotherKey"));
-    private static final Map<Integer, Long> INDEX_ENCODED_VALUES = ImmutableMap.of(1, 1337L, 2, -10L, 0, 42L);
-    private static final Function<Long, String> VALUE_MAPPER = value -> Long.toString(value - 10);
-    private static final Function<String, Long> REVERSE_MAPPER = value -> Long.parseLong(value) + 10;
+    private static final Map<DeterministicHashableWrapper<String>, Long> VALUES =
+            ImmutableMap.of(wrap("key1"), 42L, wrap("key2"), 1337L, wrap("anotherKey"), -10L);
+    private static final Set<DeterministicHashableWrapper<String>> KEYS =
+            ImmutableSet.of(wrap("key1"), wrap("key2"), wrap("anotherKey"));
 
     private final ChecksumType checksumType;
-    private IndexEncodingResult<DH<String>, Long> encoded;
 
     public IndexEncodingUtilsTest(ChecksumType checksumType) {
         this.checksumType = checksumType;
     }
 
-    @Before
-    public void setup() {
-        encoded = IndexEncodingUtils.encode(KEYS, VALUES, Function.identity(), checksumType);
-    }
-
     @Test
     public void canLookupValidChecksumType() {
         assertThat(ChecksumType.valueOf(checksumType.getId())).isEqualTo(checksumType);
     }
 
     @Test
     public void cannotLookupUnknownChecksumType() {
-        assertThatException()
-                .isThrownBy(() -> ChecksumType.valueOf(-1))
+        assertThatLoggableExceptionThrownBy(() -> ChecksumType.valueOf(-1))
                 .isInstanceOf(SafeIllegalArgumentException.class)
-                .withMessage(
-                        SafeExceptions.renderMessage("Unknown checksum type ID", SafeArg.of("checksumTypeId", -1)));
+                .hasMessageStartingWith("Unknown checksum type ID")
```

Comment:
Let's use hasLogMessage! This doesn't render the args in the string, so we can assert over the entire compile time constant message